api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java

```diff
@@ -536,9 +540,8 @@ public String submitTransaction(String signedTransaction) throws ApiException {
    */
   @Override
   public DepositParameters getDepositParameters() {
-    ProtocolParams protocolParametersFromIndexerAndConfig = ledgerDataProviderService.findProtocolParametersFromIndexerAndConfig();
-    return new DepositParameters(protocolParametersFromIndexerAndConfig.getKeyDeposit().toString(),
-        protocolParametersFromIndexerAndConfig.getPoolDeposit().toString());
+    ProtocolParams pp = protocolParamService.findProtocolParametersFromIndexerAndConfig();
```

Comment:
it's better to use the word `params` than `pp`

Reply:
It is local var in the method with three lines. In my case there will be no line feed. Let's leave it as it is...